src/prototypes/draft-email-textarea/demo/demo.scss

```diff
@@ -0,0 +1,141 @@
+@use '../../../compiled/tokens/scss/size';
+@use '../../../compiled/tokens/scss/color';
+@use '../../../compiled/tokens/scss/ease';
+@use '../../../compiled/tokens/scss/z-index';
+@use 'sass:color' as sass-color;
+@use '../../../mixins/ms';
+
+///
+/// Tooltip styles
+///
+/// These Tooltip styles will be inlined in a Gutenberg Custom HTML block
+///
+/// TODO Consider making a Tooltip patterns component in the future
+///
+
+$tooltip-color-background: var(--theme-color-background-secondary);
+$tooltip-color-border: #{sass-color.change(
+    color.$brand-primary-darker,
+    $alpha: 0.4
+  )};
+
+///
+/// 1. Horizontally centers tooltip
+/// 2. Positions tooltip vertically in "end" position (below button)
+///
+
+.c-tooltip {
+  border: ms.step(-4) solid $tooltip-color-background;
+  border-radius: size.$border-radius-medium;
+  background: $tooltip-color-background;
+  color: var(--theme-color-text-emphasis);
+  box-shadow: 0 0 0 size.$edge-medium $tooltip-color-border,
+    size.$edge-large size.$edge-large 0 size.$edge-medium #{sass-color.change(
+        color.$brand-primary-darker,
+        $alpha: 0.2
+      )};
+  inline-size: max-content; // Is this the right way to handle this?
+  inset-inline-start: 50%; // 1
+  opacity: 0;
+  position: absolute; // 1
+  z-index: z-index.$alert;
+  inset-block-start: 2.45em; // 2
```

Comment:
Instead of making this a hard value, could you position it to `100%` and then negate the border size using margin?

Example: https://codepen.io/tylersticka/pen/oNMZxBq/2c967ba0806e7ee98fec9ed5a6dfd295

Reply:
Great idea, thank you!